Installer/Setup.iss

```diff
@@ -81,6 +81,7 @@ SignTool=certum
 [Languages]
 Name: en_CA; MessagesFile: compiler:Default.isl
 Name: fr_CA; MessagesFile: compiler:Languages\French.isl
+Name: zh-CN; MessagesFile: compiler:Languages\Chinese.isl
```

Comment:
This is the reason the build fails - the language name cannot have hyphens, so you must rename it to `zh_CN` (underscore instead of hyphen).

Reply:
I'll review the rest once I have more time. Thank you! 😄